deployment/oracle/README.md

```diff
@@ -97,6 +97,8 @@ To be used with the ansible-playbook command, for example:
 
 * `--private-key=<file_name>` - if private keyfile is required to connect to the ubuntu instance.
 
+* `--user=<username>` - connect as this username
+
 ## Bridge service commands
 
 The Bridge service is named `poabridge`. Use the default `SysVinit` commands to `start`, `stop`, `restart`, and `rebuild` the service and to check the `status` of the service. 
```

Comment:
Not sure what would be required @akolotov , but may want to rename the service to `tokenbridge` to retain consistency.

Reply:
Looks like it is just a simple search-and-replace - should I do it? @akolotov

Reply:
let's create a new issue for this

Reply:
Please put the link to the issue here.